MSAL/src/public/MSALError.h

```diff
@@ -60,6 +79,26 @@ extern NSString *MSALCorrelationIDKey;
  */
 extern NSString *MSALHTTPResponseCodeKey;
 
+/*!
+ List of scopes that were requested from MSAL, but not granted in the response.
+
+ Apps normally request certain permissions that their app needs to function.
+ However, sometimes STS grants only a subset of those permissions.
```

Comment:
nit: I don't think this adds much, first line covers this.

Reply:
done